sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/samples/Sample_DocumentStatus.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.DocumentTranslation.Tests.Samples
+{
+    [LiveOnly]
+    public partial class DocumentTranslationSamples : SamplesBase<DocumentTranslationTestEnvironment>
+    {
+        [Test]
+        public void DocumentStatus()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            Uri sourceUri = new Uri("");
```

Comment:
@maririos how is the storage part going to work?

Reply:
Great question. At least for the tests, what we are trying to do is to create the Source and Target container per test, upload the documents to the Source container, run test, and let CI clean the containers after all test are done.

I wonder if that code in the samples will confuse the user or if on the contrary, it will give the user a hint of how to upload their docs. This won't work if the user already has all documents in their storage account though....

@kristapratico what are u thinking for Python?

Reply:
Hmm not sure what we'll do yet. Probably will skip running samples in CI for this first beta.

Reply:
Sounds good. We can do the same in .NET